src/Tests/InstrEngineTests/InstrEngineTests/InstrEngineTests.csproj

```diff
@@ -19,6 +19,9 @@
     <Compile Remove="EmbeddedResources\**" />
     <EmbeddedResource Include="EmbeddedResources\**" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This project should be updated to include the net70 target framework and only compile or run any of the new ref field testing for .NET7 or newer.

Reply:
That can be done in the next iteration PR, where we actually introduce the tests.